scripts/ai/AIDriveStrategyShovelSiloLoader.lua

```diff
@@ -678,7 +694,12 @@ function AIDriveStrategyShovelSiloLoader:approachTrailerForUnloading()
     local firstWpIx = course:getNearestWaypoints(self.vehicle:getAIDirectionNode())
     self:startCourse(course, firstWpIx)
     self:setNewState(self.states.DRIVING_TO_UNLOAD)
-    self.shovelController:calculateMinimalUnloadingHeight(self.targetTrailer.exactFillRootNode)
+    if self.targetTrailer.trailer["spec_pdlc_goeweilPack.balerStationary"] then
+        --- Minimal height calculation is not working for Goeweil balers
+        self.shovelController:setMinimalUnloadingHeight(2.5)
```

Comment:
Wondering if it is worth to put this into a configuration. Maybe not if it is a single vehicle...

Reply:
It is just the Göweil stationary baler.
So not worth (yet) to make a configuration for this.
Maybe if we get more tools to unload into that needs to be adjusted.